packages/loader/container-loader/src/container.ts

```diff
@@ -473,7 +473,7 @@ export class Container
 	private readonly codeLoader: ICodeDetailsLoader;
 	private readonly options: ILoaderOptions;
 	private readonly scope: FluidObject;
-	private readonly subLogger: TelemetryLogger;
+	private readonly subLogger: ITelemetryLoggerExt;
```

Comment:
Oh, right, the Ext types.  I think earlier I saw `ITelemetryLogger` plain, should probably be `ITelemetryLoggerExt`, so people can use arrays (and flat objects in next major)

Reply:
i didn't worry about moving all internal references, and some land on public interfaces where it doesn't work. where i added new public exposures i used TelemetryUTLogger. i expect someone will do all the interface clean up in next soon.

Reply:
it also changes which package the interface comes from, which i didn't want to do everywhere.